src/Instrumentation/Symfony/src/SymfonyInstrumentation.php

```diff
@@ -76,12 +76,19 @@ public static function register(): void
                     $span->setStatus(StatusCode::STATUS_ERROR, $exception->getMessage());
                 }
                 if ($response) {
-                    if ($response->getStatusCode() >= 400) {
+                    if ($response->getStatusCode() >= Response::HTTP_BAD_REQUEST) {
                         $span->setStatus(StatusCode::STATUS_ERROR);
                     }
                     $span->setAttribute(TraceAttributes::HTTP_STATUS_CODE, $response->getStatusCode());
                     $span->setAttribute(TraceAttributes::HTTP_FLAVOR, $response->getProtocolVersion());
-                    $span->setAttribute(TraceAttributes::HTTP_RESPONSE_CONTENT_LENGTH, $response->headers->get('Content-Length'));
+                    $contentLength = $response->headers->get('Content-Length');
+                    /** @psalm-suppress PossiblyFalseArgument */
+                    //BinaryFileResponse and StreamedResponse return boolean as response
+                    if (null === $contentLength && is_string($response->getContent())) {
```

Comment:
Don't know if that may happen, but what in the case when `Content-length` header is not provided and `$response->getContent()` returns false ? 
Otherwise LGTM.

Reply:
it's set null as before - default value for `$response->headers->get()`, I added tests that cover this case and also removed some redundant parameters.

Reply:
I missed test. I was just thinking if it makes sense to set this attribute in this case